### ISSUE TITLE
MSSQL SQL syntax incompatibilities

### ISSUE THREAD

**Issue by yivo** [openware/peatio#207]
```
app/models/account.rb:    self.class.connection.execute "update accounts set balance = balance + #{delta_b}, locked = locked + #{delta_l} where id = #{id}"
spec/models/account_spec.rb:      ActiveRecord::Base.connection.execute "update accounts set balance = balance + 3, locked = locked - 8 where id = #{subject.id}"
spec/models/account_version_spec.rb:        ActiveRecord::Base.connection.execute "update accounts set balance = balance + 3, locked = locked - 8 where id = #{account.id}"
spec/models/account_version_spec.rb:        ActiveRecord::Base.connection.execute "update accounts set balance = balance + 3, locked = locked - 8 where id = #{account.id}"
```

**Comment by yivo** [comment]
Please base your branch on master and target your PR to branch devel. Push PR as you start working.

**Comment by yivo** [comment]
Closed via #567.
